README.md

```diff
@@ -1,2 +1,36 @@
-# Math_Magicians
-???///???
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio desktop version
```

Comment:
- [ ] Please kindly update update your `readme file` with information about this particular milestone, your `readme file` is still talking about your portfolio project, please kindly update it, as this will make your readme file more informative and professional.